splice_access_api/src/main/java/com/splicemachine/access/configuration/SQLConfiguration.java

```diff
@@ -210,6 +210,9 @@
     public static final String NESTEDLOOPJOIN_BATCH_SIZE = "splice.nestedLoopJoin.batchSize";
     private static final int DEFAULT_NESTEDLOOPJOIN_BATCH_SIZE = 10;
 
+    public static final String CONTROL_EXECUTION_ROWS_LIMIT = "splice.controlExecution.rowsLimit";
+    private static final int DEFAULT_CONTROL_EXECUTION_ROWS_LIMIT = 2000000;
+
```

Comment:
When we start to run in control mode I think it makes sense to continue doing so until we start having memory issues. This size is in line with the maximum broadcast size, so I think it's ok. What size would you suggest?